### ISSUE TITLE
Error with node-sass 2.0.0

### ISSUE THREAD

**Issue by ocombe** [brunch/sass-brunch#68]
Hi, you will have to update your plugin to node-sass 2.0.0 (still in beta), it's broken.
Here is the error log:

```sh
brunch w

C:\[...]\node_modules\brunch\lib\fs_utils\pipeline.js:78
          throw new Error("Brunch SourceFile: file " + path + " data is invali
                ^
Error: Brunch SourceFile: file admin\admin-app.scss data is invalid
  at C:\[...]\node_modules\brunch\lib\fs_utils\pipeline.js:78:17
  at libsass.render.success (C:\[...]\node_modules\sass-brunch\index.js:96:7)
  at options.success (C:\[...]\node_modules\node-sass\lib\index.js:169:7)
```

**Comment by es128** [comment]
Unable to reproduce with the latest plugin version using node-sass 1.2.3.

What does the content of the source file causing the error look like?

**Comment by ocombe** [comment]
That's the thing, I manually changed the version of node sass to the latest (2.0.0-beta), it works fine with 1.2.3.
I'm just warning you that you will have to update the lib for the next release of node-sass (that includes updates for the source maps as well) :)

**Comment by es128** [comment]
Ah ok, I understand. They probably changed the API along with the major version bump. Not really an issue, but thanks for the heads up.

**Comment by EvanMcEwen** [comment]
Any chance this will be fixed soon? They are on 2.0.1 release now and it still isn't working.

**Comment by es128** [comment]
@EvanMcEwen It was fixed with the last release (a couple days ago). If you're having problems with it please open a new issue.
